### PR TITLE
Fix right panel for peeking rooms

### PR DESCRIPTION
**Pull request by t3chguy**
Fixes https://github.com/vector-im/element-web/issues/15349

image

### CODE REVIEW

File: src/stores/WidgetStore.ts [hunk 1]
```diff
@@ -118,6 +118,7 @@ export default class WidgetStore extends AsyncStoreWithClient<IState> {
     }
 
     private loadRoomWidgets(room: Room) {
+        if (!room) return;
```
Comment (turt2live) [inline src/stores/WidgetStore.ts:121]: would be good to link to https://github.com/vector-im/element-web/issues/8933 as the user is a guest when peeking

**Review by turt2live — APPROVED** [review]
otherwise lgtm

File: src/stores/WidgetStore.ts (hunk 1 above)
Comment (t3chguy) [reply src/stores/WidgetStore.ts:121]: I don't think this null-guard has anything to do with that however, given that peeking guests have valid rooms too.
I saw a `getRooms()` call return `[null]` which is what I am guarding against here.

Comment (turt2live) [reply src/stores/WidgetStore.ts:121]: uh, that sounds like a larger problem that shouldn't be resolved in the depths of the widget code - that'll break the room list and pretty much everything else too.

Comment (t3chguy) [reply src/stores/WidgetStore.ts:121]: Indeed, I hit it once and could not repro the next 10x times. This is the only error log that came of it however.

**Pull request merged**